app/logic/resultsets.py

```diff
@@ -177,7 +177,7 @@ def is_approximatable_constant(input_evaluated):
             not input_evaluated.free_symbols and
             not input_evaluated.is_Integer and
             not input_evaluated.is_Float and
-            input_evaluated.is_finite is not True)
+            not input_evaluated.is_infinite)
```

Comment:
Hmm. As far as I can see, though, Travis is still failing, because it can't import `mpmath`, even though you've added the dependency. Odd.